ocaml/xcp-rrdd/bin/rrdd/xcp_rrdd.ml

```diff
@@ -390,6 +403,18 @@ let dss_hostload xc domains =
         ~value:(Rrd.VT_Float load_per_cpu) ~min:0.0 ~ty:Rrd.Gauge ~default:true
         ()
     )
+  ; ( Rrd.Host
+    , Ds.ds_make ~name:"running_vcpus" ~units:"(fraction)"
+        ~description:"The total number of running vCPUs per host"
+        ~value:(Rrd.VT_Int64 (Int64.of_int load))
+        ~min:0.0 ~ty:Rrd.Gauge ~default:true ()
+    )
+  ; ( Rrd.Host
+    , Ds.ds_make ~name:"running_domains" ~units:"(fraction)"
```

Comment:
From the comment, this is a fraction. But from the calculation, this seems a count number?

Reply:
I could convert fraction -> int.
However, I still have a question. Even though I've set the type of this metric as int64, I'm still receiving decimal values, such as 1.333, when I query it. Is this due to the final display of this metric being the calculated average over a time range？

Reply:
I understand the RRD will return a consolidated value:
https://docs.xenserver.com/en-us/xenserver/8/developer/sdk-guide/using-http.html#getting-xenserver-operational-metrics
You could try to change the Consolidation Functions (CF) to confirm this. By default it is `average`.

Reply:
Here I think the `units` requires a string of unit. Like "memory_total_kib" with unit "KiB", how about "running_domains" with unit "domains"?

Reply:
Agree, I will use `domain` `vcpus` as the units.